packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -63,6 +63,8 @@ declare global {
     curatedAfter?: Date|string|null,
     timeField?: keyof DbPost,
     postIds?: Array<string>,
```

Comment:
Surely if we were starting from scratch, we'd make this do the behavior of `exactPostIds`. I suspect the risk is of breaking something is worth the reduction of another option here. You could ask any of the devs if they have any memory of something that that would break?

Reply:
I think the root issue here is that events are excluded in the default view (in a way that's quite hard to unexclude). IMO it would be better to exclude them in all the views that don't want them (via an `excludeEvents` param that behaves more as expected, the current one doesn't really work because posts with a groupId are also excluded elsewhere)